### ISSUE TITLE
Server crashes immediately when player joins

### ISSUE THREAD

**Issue by QyInvoLing** [TartaricAcid/ysm#93]
I've set up a 1.20.1 fabric server with only one mod: ysm 1.1.5, everything keeps normal until any player joins the server, which caused a crash.
crash report: 
```
---- Minecraft Crash Report ----
// Shall we play a game?

Time: 2024-04-09 11:50:35
Description: Exception in server tick loop

java.lang.NoSuchFieldError: OooOo0oOoOO0o00OO000oO00
	at com.elfmcys.yesstevemodel.OoOoOOO00OoOo00oooOoo00.<clinit>(Unknown Source)
	at net.minecraft.class_1657.OooOo0oOoOO0o00OO000oO00(class_1657.java:11888)
	at com.elfmcys.yesstevemodel.OoOOOoO0ooo0oO000O0o0O.OooOo0oOoOO0o00OO000oO00(Unknown Source)
	at com.elfmcys.yesstevemodel.ooOoO0OOooO00oOo00oooOoo.OooOo0oOoOO0o00OO000oO00(Unknown Source)
	at com.elfmcys.yesstevemodel.Ooo00OOoOO0oOOoO000o0oo0.OooOo0oOoOO0o00OO000oO00(Unknown Source)
	at net.fabricmc.fabric.api.networking.v1.ServerPlayConnectionEvents.lambda$static$2(ServerPlayConnectionEvents.java:48)
	at net.fabricmc.fabric.impl.networking.server.ServerPlayNetworkAddon.onClientReady(ServerPlayNetworkAddon.java:66)
	at net.minecraft.class_3324.handler$zpl000$fabric-networking-api-v1$handlePlayerConnection(class_3324.java:5018)
	at net.minecraft.class_3324.method_14570(class_3324.java:215)
	at net.minecraft.class_3248.method_33800(class_3248.java:131)
	at net.minecraft.class_3248.method_14384(class_3248.java:118)
	at net.minecraft.class_3248.redirect$zpm000$fabric-networking-api-v1$handlePlayerJoin(class_3248.java:563)
	at net.minecraft.class_3248.method_18784(class_3248.java:68)
	at net.minecraft.class_2535.method_10754(class_2535.java:259)
	at net.minecraft.class_3242.method_14357(class_3242.java:172)
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:908)
	at net.minecraft.class_3176.method_3813(class_3176.java:283)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:824)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:671)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:265)
	at java.base/java.lang.Thread.run(Thread.java:840)

```

**Comment by Mrpressed** [comment]
Had this same issue 1.20.1 fabric

**Comment by kotisoff** [comment]
Same

**Comment by ClintFlames** [comment]
same for 1.19.2

**Comment by WestonQingYing** [comment]
Same

**Comment by TartaricAcid** [comment]
Fix in 2.2.1
